FirstClasses/average/src/average.erl

```diff
@@ -3,4 +3,49 @@
 -export([average/1]).
 
 average(_List) ->
```

Comment:
I know I left `_Variable` in quite a few places but it was just not to have too many compiler warnings. It is usually common practice to use variables without it like `List` instead od `_List`.